src/config.cc

```diff
@@ -449,26 +449,17 @@ void Config::initFieldCallback() {
       }},
       {"rocksdb.max_bytes_for_level_base", [this](Server* srv, const std::string &k, const std::string& v)->Status {
         if (!srv) return Status::OK();
-        if (!RocksDB.level_compaction_dynamic_level_bytes) {
-          return Status(Status::NotOK, errNotSetLevelCompactionDynamicLevelBytes);
-        }
```

Comment:
Why do you make these changes?

Reply:
My bad, this change should be in #610 pull request, I'll clean this change.